### PR TITLE
Build task bundle in release branch

### PR DESCRIPTION
**Pull request by joejstuart**
Build the verify-enterprise-contract task bundle on pull request and push. Also, modify the patch script to support the bundle builds.

https://issues.redhat.com/browse/EC-428

### CODE REVIEW

File: hack/patch-release-pipelines.sh [hunk 1]
```diff
@@ -32,41 +32,47 @@ fi
...
-  fi
+  for comp in ${components[@]}; do
+    main_pipeline=".tekton/${comp}-${main_release_name}-$p.yaml"
+    release_pipeline=".tekton/${comp}-${release_name/./}-$p.yaml"
 
-  # Loop over each commit
-  for sha in $changes; do
-    echo "Applying changes from commit $(git log -n1 --pretty="'%h %s'" $sha)"\
-      "to pipeline definition file '$release_pipeline'"
+    if [[ "$digest_bumps" != "digest_bumps" ]]; then
```
Comment (zregvart) [inline hack/patch-release-pipelines.sh:45]: (comment) I'm a bit worried that when the commit messages change we'll miss some commits here, something to keep in mind is to keep updating those messages for `--grep` below

**Review by zregvart — APPROVED** [review]
I didn't really focus on reviewing the YAML files, the build passed so that what's important.

We get PRs like #1372, do we need the `digest_bumps` if those keep flowing?

**Comment by simonbaird** [comment]
Notes:
* I thought the `*.alpha.yaml` files would only be present in the corresponding release branch..?
* The file name is long, just wondering if "task" would be enough of a distinguisher?

File: hack/patch-release-pipelines.sh (hunk 1 above)
Comment (simonbaird) [reply hack/patch-release-pipelines.sh:45]: Yeah, a more robust detection of the different commits would be nice. Not a blocker for this PR of course.

File: hack/patch-release-pipelines.sh [hunk 2]
```diff
@@ -32,41 +32,47 @@ fi
...
+      # Stage the changes
+      git add $release_pipeline
 
-  # Let's keep the main branch pipeline for now since it is useful to diff against
-  #git rm $main_pipeline
+    done
+
+  done
+    # Let's keep the main branch pipeline for now since it is useful to diff against
+    #git rm $main_pipeline
```
Comment (simonbaird) [inline hack/patch-release-pipelines.sh:75]: Even though it's commented out, I think maybe this belongs inside the "done" on line 73.

**Review by simonbaird — APPROVED** [review]

**Comment by joejstuart** [comment]
@simonbaird @zregvart I've made some changes since you approved, if you wanna take another look before I merge. https://github.com/enterprise-contract/ec-cli/pull/1377/commits/abd81cebad761159751ec43c0a8cca1ea4ed4f0a

**Pull request merged**